server/routerlicious/packages/memory-orderer/src/localOrderer.ts

```diff
@@ -138,6 +141,7 @@ export class LocalOrderer implements IOrderer {
 
     public scriptoriumLambda: LocalLambdaController | undefined;
     public foremanLambda: LocalLambdaController | undefined;
+    public museLambda: LocalLambdaController | undefined;
```

Comment:
We should change the name muse. Maybe Moira or Clotho?

Reply:
Moira should be easier to pronounce :)